package.json

```diff
@@ -39,6 +39,7 @@
     "@vitest/coverage-istanbul": "^1.6.0",
     "bumpp": "^9.4.1",
     "conventional-changelog-cli": "^5.0.0",
+    "core-js": "^3.37.1",
```

Comment:
May not put it in root?

Reply:
Maybe upstream bug/design 🤔 Farm currently starts resolving `core-js` from each dependency's  `node_modules`, which is a problem when using Monorepo. So put `core-js` in the root package to ensure `core-js` can be resolved and not unnecessarily increase the number of dependencies on other packages.